projects/kubernetes/kubernetes/Makefile

```diff
@@ -11,20 +11,20 @@ CLONE_URL=https://github.com/$(COMPONENT).git
 AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 AWS_REGION?=us-west-2
 
-BASE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+KUBE_BASE_REPO?=$(IMAGE_REPO)
```

Comment:
Does this work if image repo is defined below instead of above?

Reply:
Good call.Fixed